Dockerfile

```diff
@@ -2,9 +2,7 @@ FROM golang:alpine AS builder
 
 ADD . $GOPATH/src/github.com/aerospike/aerospike-prometheus-exporter
 WORKDIR $GOPATH/src/github.com/aerospike/aerospike-prometheus-exporter
-RUN apk add git \
```

Comment:
Are you sure `git` is already present on the target image?

Reply:
`git` package is not required anymore since we removed `go get` from `Dockerfile`. Works!